src/main/java/com/gluonhq/substrate/model/PrivateProjectConfiguration.java

```diff
@@ -50,25 +49,19 @@
  * If this method has not been called, getJavaStaticLibsPath() will return the default location, taking into account
  * the value of javaStaticSdkVersion. If that value is not set, the default value is used.
  */
-public class ProjectConfiguration {
+public class PrivateProjectConfiguration {
```

Comment:
I don't like the class name using `Private`, maybe `InternalProjectConfiguration`?

Reply:
Done